dask/dataframe/tests/test_dataframe.py

```diff
@@ -2003,6 +2003,26 @@ def test_columns_assignment():
     eq(df, ddf)
 
 
+def test_attribute_assignment():
+    df = pd.DataFrame({'x': [1, 2, 3, 4, 5],
+                       'y': [1., 2., 3., 4., 5.]})
+    ddf = dd.from_pandas(df, npartitions=2)
+
+    ddf.y = ddf.x + ddf.y
+    eq(ddf, df.assign(y=df.x + df.y))
+
+
+def test_inplace_operators():
+    df = pd.DataFrame({'x': [1, 2, 3, 4, 5],
+                       'y': [1., 2., 3., 4., 5.]})
+    ddf = dd.from_pandas(df, npartitions=2)
+
+    ddf.y **= 0.5
```

Comment:
Yes, it does. Though if you wanted to support *in-place* assignment you would have to implement `__ipow__`.
For example:
```
>>> a = np.linspace(0,10,5)
>>> a
array([  0. ,   2.5,   5. ,   7.5,  10. ])
>>> a.__ipow__(0.5)
array([ 0.        ,  1.58113883,  2.23606798,  2.73861279,  3.16227766])
>>> a
array([ 0.        ,  1.58113883,  2.23606798,  2.73861279,  3.16227766])
```

(notice how `__ipow__` returns the result as `__pow__` does: what makes the operation really in-place is that `__ipow__` can return self instead of a copy, which it doesn't *have* to)